rfc6962-bis.xml

```diff
@@ -510,186 +596,170 @@ Section 8.2 of <xref target="RFC3447"/>) using a key of at least 2048 bits.
         opaque sct_extension_data&lt;0..2^16-1&gt;;
     } SctExtension;
 
-    SctExtension SctExtensions&lt;0..2^16-1&gt;;
-
     struct {
-        Version sct_version;
         LogID log_id;
         uint64 timestamp;
-        SctExtensions extensions;
+        SctExtension sct_extensions&lt;0..2^16-1&gt;;
         digitally-signed struct {
-            Version sct_version;
-            SignatureType signature_type = certificate_timestamp;
-            uint64 timestamp;
-            LogEntryType entry_type;
-            select(entry_type) {
-                case x509_entry: CertInfo;
-                case precert_entry_V2: CertInfo;
-            } signed_entry;
-            SctExtensions extensions;
-        };
-    } SignedCertificateTimestamp;</artwork>
+            TransItem timestamped_entry;
```

Comment:
I wanted to ensure that the definition of TransItem left open the possibility that V1 structures could be accommodated.  It seemed like something that we might need, and I figured it was easier to take it out (if we decide we don't need it) than to try to bolt it on later.

Personally, I think the gain is worth the (complexity) pain.  RFC6962-compliant logs aren't going to go away overnight.  This mechanism lets us deliver V1 inclusion proofs.  It could be used for gossiping V1 structures.